java-checks/src/main/java/org/sonar/java/filters/LombokFilter.java

```diff
@@ -181,12 +182,34 @@ public void visitAssignmentExpression(AssignmentExpressionTree tree) {
   }
 
   private static boolean usesAnnotation(ClassTree classTree, List<String> annotations) {
+    return usesAnnotation(classTree, annotations, !CHECK_LOCAL_NAME);
```

Comment:
When calling `usesAnnotation` with 2 parameters, then we should expect a predictable behavior where the value of 3rd parameter should always be `true` or always be ` false`.
```suggestion
    return usesAnnotation(classTree, annotations, false);
```